### DISCUSSION TITLE
Request for a bit more feature complete sample to learn from.

### DISCUSSION THREAD

**Discussion by Laro88** [uNetworking/uWebSockets#1141] [Q&A]
I am struggling to understand the filosophy in the design, please bear with me :-)

So far i have experimented with the samples, however I haven't found one that demonstrates how publish to specific clients / all clients.

So I would like a bit more feature complete example handling multiple clients, and demonstrating the way to publish/send from other threads. Generous comments and recommendations would be appreciated :-)

**Comment by Laro88** [comment]
I got a mail that asked if I have looked in the doc and at an specific example, and yes I have :-)
However I found no examples of the ->defer( which seems to be needed when doing cross thread publishing to specific / all clients, and I could also use some more description on how to make best use of this library :-)

**Comment by ghost** [comment]
You have more examples in the JavaScript repo and there is a read more link with basic user manual

**Comment by totaland** [comment]
Hi @alexhultman is there someone else that you can ask to help provide a full-stack example? Just in case you are too busy to do it? Surely, you have written some test projects that you can show us?
